Sources/Gui/Main.cpp

```diff
@@ -57,13 +57,23 @@
 #endif
 
 
+#ifdef _MSC_VER>=1900	// Visual Studio 2015 or higher
+extern "C" { FILE __iob_func[3] = { *stdin,*stdout,*stderr }; }
```

Comment:
Yes, sorry, I cloned the repo again and forgot to include the dlls. [This](http://puu.sh/nozm9/640ad631ac.png) is the actual error.
But it turns out if I include the 2 images at compile time (with `#include` like it was) I do get [errors](http://puu.sh/nozA7/77a5d6302b.png) inside VS but the project actually builds... When I run it I get [this](http://puu.sh/nozBO/9127321d7e.png) error (SystemMessages.log is empty) Why could it fail to build the scripts?.

I never imagined it would be *this* difficult...